config/config.default.js

```diff
@@ -63,6 +63,10 @@ var path = require('path'),
             defaultConnectionRouter: 'basic3' //'basic', 'basic2', 'basic3'
         },
 
+        core: {
```

Comment:
why is this called core? shouldn't we use 'cosntraints' and 'enableCustom' instead?